pandas/tseries/tests/test_partial_string_indexes.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+
+import pandas.util.testing as tm
```

Comment:
instead of creating a new files, these should go with the existing tests in test_timeseries.py

further, don't create new ways of testing, just follow along the existing.